core/controllers/ingest.py

```diff
@@ -39,7 +39,7 @@
 from core.messaging.messaging import failures_to_messages, group_validation_messages
 from core.metrics import capture_ingest_metrics
 from core.table_extraction.config.pf_r1_config import PF_TABLE_CONFIG
-from core.transformation.pathfinders.pf_transform_r1 import pathfinders_transform
+from core.transformation.pathfinders.pf_transform_r1 import transform as pf_r1_transform
```

Comment:
This reads maybe slightly weird to me - we have a module called `pf_transform_r1` and then a function aliased as `pf_r1_transform`. The overlap and inconsistency feels weird just as I'm reading this. I don't really have any strong objections or suggestions though.

Reply:
My logic was basically that a function name should end with a verb, whereas a module name needn't - however I agree it does read more like I've just got confused haha